index.html

```diff
@@ -231,13 +224,6 @@
       manufacturer. They may also expose an <a>NDEF</a> message.
     </p>
   </div>
-  <p>
```

Comment:
This is informative content and could stay IMHO. We could add a note that given (link) reasons, we don't support P2P at the moment.

Reply:
Done.